servers/apps/matterbridge/README.md

```diff
@@ -0,0 +1,50 @@
+# Instructions for bridging if topic as group is used:
+
+Prerequisites:
+
+- Access to falcon server of FOSS United
+- Access to the @fu-bot:tchncs.de account on matrix
```

Comment:
I don't know who all have access to these credentials - I definitely don't have them and they're not stored on the FOSS United vault from what I can see.

Reply:
I think I gave access to ansh and mangesh, not sure who/where I have to give the credentials to exactly